tests/Messenger/Transport/AzureTransportTest.php

```diff
@@ -553,6 +554,44 @@ public function testSendWithBrokerPropertiesStampGetsConvertedToHttpHeader(): vo
         $transport->send($envelope);
     }
 
+    /**
+     * The BrokerProperties stamp must be used to generate a BrokerProperties HTTP header
+     */
+    public function testSendWithBrokerPropertiesStampAndDelayStamp(): void
+    {
+        $envelope = new Envelope(new class {}, [
+            new AzureBrokerPropertiesStamp(), new DelayStamp(50000)
```

Comment:
The `AzureBrokerPropertiesStamp` should be removed here as it should automatically be added.